benchmark/size/report.md

```diff
@@ -1,57 +1,57 @@
 | Package | Version | Publish Size | browser:Webpack | browser:Rollup | browser:EsBuild |
 | :------ | :------ | :----------- | :------ | :----- | :------- |
-|@aws-sdk/abort-controller|3.109.0|43.8 KB|✅(5.62.1)|✅(2.59.0)|✅(0.13.12)|
```

Comment:
this means the benchmark stopped working around this version for rollup-tested modules.